src/tracker/CMakeLists.txt

```diff
@@ -8,6 +8,11 @@ set(PSMOVEAPI_TRACKER_PLATFORM_SRC)
 # Build the OpenCV-based camera tracking module? (see psmove_tracker.h)
 option(PSMOVE_BUILD_TRACKER "Build the Tracker module (needs OpenCV)" ON)
 
+# The MSVC instructions use a locally built OpenCV. So, we need to add it to the path here when building using MSVC.
+IF(MSVC)
+	set(OpenCV_DIR ${ROOT_DIR}/external/opencv/build CACHE PATH "Path to local OpenCV source tree" FORCE)
```

Comment:
Could we put this into the MSVC instructions as `-DOpenCV_DIR=../external/opencv/build/` parameter to cmake? This would allow the instructions to work correctly while still not hardcoding a path there.